apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,23 @@
 <div
+[style]="{ 'background-color': color }"
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
-  [class]="customClass">
+  [class]="color">
   <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
+    [src]="imageUrl"
     width="200px" />
 
   <section>
+    @for (item of list; track $index) {
     <app-list-item
-      *ngFor="let item of list"
-      [name]="item.firstname"
+      [name]="item.name"
```

Comment:
This is not very customizable. What will you do if you need to add an icon but only for one card, or a new button ? Look at ngTemplateOutlet